### PR TITLE
Sellingpage mobile and search

### PR DESCRIPTION
**Pull request by Danryg**
Fixed Search in selling page, mobile responsive layout and are you sure modal component

### CODE REVIEW

**Comment by emilsvennesson** [comment]
Can you rename the folders to lowercase? `sell` and `wizard` or something like that.

File: app/src/components/ConfirmationModal.tsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+const style = {
+  position: 'absolute' as 'absolute',
+  top: '50%',
+  left: '50%',
+  transform: 'translate(-50%, -50%)',
+  width: '400px',
+};
+
+export default function ComfirmationCard({
```
Comment (Kexon) [inline app/src/components/ConfirmationModal.tsx:29]: ConfirmationCard

File: app/src/pages/SellingPage/ConditionCheckCard.tsx [hunk 2]
```diff
@@ -80,17 +81,18 @@ export default function ConditionCheckCard({
...
       </FormControl>
       <TextField
         disabled={!active}
         fullWidth
         multiline
         rows={4}
-        maxRows={4}
         value={describtion}
         onChange={handleDescribtionChange}
         label="Beskriv kvaliten på boken"
```
Comment (Kexon) [inline app/src/pages/SellingPage/ConditionCheckCard.tsx:98]: kvalitén tror jag

File: app/src/pages/SellingPage/ConditionCheckCard.tsx [hunk 3]
```diff
@@ -33,11 +33,11 @@ export default function ConditionCheckCard({
   disabled = false,
   active,
 }: ConditionCheckCardProps) {
-  const [condition, setCondition] = React.useState(conditions.good);
+  const [condition, setCondition] = React.useState(BookCondition.GOOD);
   const [describtion, setDescribtion] = React.useState('');
```
Comment (Kexon) [inline app/src/pages/SellingPage/ConditionCheckCard.tsx:37]: description, men det är typ skitsamma

File: app/src/pages/SellingPage/wizard/SetPriceWindow.tsx [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+          {/* Continue button */}
+          <Box flexGrow={1} flexShrink={20}>
+            <Button
+              onClick={() => handleNext()}
+              variant="contained"
+              size="large"
+              disabled={!active}
+            >
+              Publisera
```
Comment (Kexon) [inline app/src/pages/SellingPage/wizard/SetPriceWindow.tsx:66]: Publicera

File: app/src/components/ConfirmationModal.tsx (hunk 1 above)
Comment (Kexon) [reply app/src/components/ConfirmationModal.tsx:29]: användes denna komponenten ngnstans?

**Pull request merged**